tests/utils.js

```diff
@@ -738,6 +739,20 @@ module.exports = {
       .catch(done.fail);
   },
 
+  seedTestDataNative: async (userContactDoc, documents) => {
+    try {
+      documents.forEach(async doc => await module.exports.saveDocNative(doc));
+    } catch (err) {
+      console.log('error saving docs');
+    }
```

Comment:
If this fails, is it desirable to continue on? I assume that if this breaks, all bets are off and something else will fail later, but it'll be harder to debug since "later" we'll end up getting an error about some element missing on the page when the real problem was that some seed data never got stored. If that's the case, please don't catch the error, but instead let it blow up right here so we can more easily track down the problem.